core/frontend/src/components/autopilot/EndpointManager.vue

```diff
@@ -9,7 +9,10 @@
         width="80%"
         class="my-4"
       />
```

Comment:
We should also conditionally show this divider

Reply:
fixed